### PR TITLE
[IMP] rename_modules: rename rating mixin parent_res_model as well

### PR DESCRIPTION
**Pull request by duong77476**
If not , rating feature will be gone for good

### CODE REVIEW

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -1079,6 +1079,12 @@ def rename_models(cr, model_spec):
                     "UPDATE mail_activity SET res_model=%s where res_model=%s",
                     (new, old),
                 )
+        if is_module_installed(cr, "rating"):
```
Comment (legalsylvain) [inline openupgradelib/openupgrade.py:1082]: Better check if table existe.

**Review by legalsylvain — APPROVED** [review]

File: openupgradelib/openupgrade.py (hunk 1 above)
Comment (duong77476) [reply openupgradelib/openupgrade.py:1082]: @legalsylvain ok as you wish sir

Comment (duong77476) [reply openupgradelib/openupgrade.py]: @pedrobaeza Okay, done sir, i saw the above check both so i copy :v

Comment (pedrobaeza) [reply openupgradelib/openupgrade.py]: Yes, it's true that the module installed check is used up on the code for `mail`. I suppose that one was for not having to check a lot of table existence and as the module name is expected to be constant. In this case, I think one check is enough.

Comment (duong77476) [reply openupgradelib/openupgrade.py]: Okay, that 's fine by me

**Pull request merged**